packaging/chocolatey/crc/crc.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>crc</id>
+    <version>2.11.0</version>
+    <owners>https://github.com/crc-org</owners>
+    <title>OpenShift Local</title>
+    <authors>anjannath</authors>
+    <projectUrl>https://crc.dev/blog/about</projectUrl>
+    <iconUrl>https://i.ibb.co/m0b6dsL/crc-upstream-logo.png</iconUrl>
+    <licenseUrl>https://github.com/crc-org/crc/blob/main/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <tags>crc ocp openshift openshiftlocal</tags>
+    <summary>CRC brings a minimal OpenShift Container Platform 4 cluster and Podman container runtime to your local computer</summary>

```

Comment:
Wondering if this should be 'Red Hat OpenShift Container Platform 4' rather than "OpenShift Container Platform 4"